stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -4,95 +4,114 @@ This package provides the functionality required to access and manipulate data s
+To access a database, you must first create an `endpoint`, which is a virtual representation of the physical endpoint that you are trying to connect to. Create an endpoint of the JDBC client type (i.e., `jdbc:Client`) and provide the necessary connection parameters. This will create a pool of connections to the specified database. A sample for creating an endpoint with JDBC client can be found below. 
 
 **NOTE**: Even though JDBC client type supports connecting to any type of relational database that is accessible via JDBC, if you are using a MySQL or H2 database, it is recommended to use endpoints that are created using the client types specific to them via the relevant ballerina packages. 
 
 ### Database operations
 
-Once the endpoint is created, database operations can be executed through that endpoint. This package provides support for creating tables and executing stored procedures. It also supports selecting, inserting, deleting, updating, and batch updating data. Samples for these operations could be found below. Details of the SQL data types and query parameters relevant for these database operations could be found in the documentation of SQL package. Once the operation is completed, `stop` function must be called to terminate the connection pool of the endpoint. 
+Once the endpoint is created, database operations can be executed through that endpoint. This package provides support for creating tables and executing stored procedures. It also supports selecting, inserting, deleting, updating, and batch updating data. Samples for these operations could be found below. Details of the SQL data types and query parameters relevant for these database operations could be found in the documentation of SQL package. 
 
 ## Samples
 
 ### Creating an endpoint
 ```ballerina
 endpoint jdbc:Client testDB {
-    url:"jdbc:hsqldb:file:./target/tempdb/TEST_SQL_CONNECTOR",
-    username:"SA",
-    poolOptions:{maximumPoolSize:1}
+    url: "jdbc:mysql://localhost:3306/testdb",
+    username: "root",
+    password: "root",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
 };
 ```
 The full list of endpoint properties can be found in the `sql:PoolOptions` type.
 
 ### Creating tables
 
-Following is an example of creating a table with two fields. One field is of type int and the other of varchar. The CREATE statement is executed via the `update` operation of the endpoint.
+Following is an example of creating a table with two columns. One field is of type int and the other of varchar. The CREATE statement is executed via the `update` operation of the endpoint.
```

Comment:
The first sentence says "columns" but the next sentence and the code comment says field/s